shared/css/phase1-design-system.scss

```diff
@@ -173,6 +173,15 @@ p {
       }
     }
   }
+  .infoIcon {
```

Comment:
Yeah, that's one of the `!important` cause but the `!important` (s) in the original code.
I feel like it's still necessary unless original code was changed.

Though, I feel like using `FontAwesomeV6Icon` and/or updating original code/styles might eliminate the need of `!imporant` here

Reply:
Thanks Denys! I updated to `FontAwesomeV6Icon` but it did not eliminate the need of `!important`.
Also - what do you mean by 'updating original code/styles'? Which code/styles are you referring to?

Reply:
By original code I've meant the `application.scss `styles.
Checked locally from your branch. Interesting, looks like that original `!important` might be already removed, meaning you/we can try removing these two `!important` from line `176` and `172` and keep it without `!important` if nothing will be broken. Might be better to do it as a separate followup pr so that it'll be easier to revert it in case of anything

Reply:
also, in case we won't need this `!important`, it might be worth moving icon color styles into `apps/src/templates/projects/personal-projects-name-cell.module.scss` (try using nested scss selector in case of anything)

Reply:
Yay! I moved the icon color style rules back to `personal-projects-name-cell.module.scss` where I originallyhad them. Thanks Denys!
Here's a separate PR to remove `!important` from the icon color styles in `phase1-design-system.scss`. Thank you! https://github.com/code-dot-org/code-dot-org/pull/57023

Reply:
@levadadenys now that `staging` is updated with https://github.com/code-dot-org/code-dot-org/pull/57023 (thanks for your help with it!), could you take another look at this PR since it's changed a bit since you last approved? Functionality has not changed. Thanks!